build_bundle_image.sh

```diff
@@ -330,6 +340,15 @@ function prepare_temp_directory {
 
 	local latest_tomcat_version=$(get_latest_tomcat_version "${tomcat_version}")
 
+	if [ -z "${latest_tomcat_version}" ]
+	then
+		lc_log ERROR "Unable to retrieve the latest Tomcat version."
+
+		exit "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+	else
+		lc_log INFO "Latest Tomcat version set to ${latest_tomcat_version}."
+	fi
+
```

Comment:
@kiwm, I'm thinking better know and I believe we should not stop the script to proceed here, instead we can only display a warning in line `198` and let the Docker image be built with the actual Tomcat version or the version available on master (file `app.server.properties`).

Does it make sense to you?